gateway-ha/pom.xml

```diff
@@ -205,7 +208,7 @@
 
         <dependency>
             <groupId>io.dropwizard</groupId>
-            <artifactId>dropwizard-logging</artifactId>
+            <artifactId>dropwizard-request-logging</artifactId>
```

Comment:
would be nice to get rid of this and just use jetty directly .. but for now it might have to stay